src/Entity/ApiToken.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ApiTokenRepository;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: ApiTokenRepository::class)]
+#[ORM\Table(name: "user_api")]
+class ApiToken
+{
+    #[ORM\Id]
+    #[ORM\Column(type: 'string', length: 50)]
+    private $userId;
+
+    #[ORM\Column(name: 'key', type: 'string', length: 50)]
+    private $apiToken;
+
+    #[ORM\Column(name: 'expiration_date', type: 'datetime_immutable', nullable: true)]
```

Comment:
I thought we said that it shouldn't be immutable... am i missing something rn?

Reply:
We did :) However, I failed to made the commit and push yesterday.